### ISSUE TITLE
Query params serialization in cy.request method.

### ISSUE THREAD

**Issue by jonyr** [cypress-io/cypress-documentation#3937]
## Query params serialization in cy.request method.

```
[X] Bug report
[ ] Content update
[ ] Process update (build, deployment, ... )
```

## Unexpected behavior

Hi everyone, I'am using Cypress to create an integration tests suites for a Restful API.

I need to make a GET the following request to the API Backend

`/api/products?category=fruit&category=meat` and is not valid doing it as `/api/products?category[]=fruit&category[]=meat` or `/api/products?category=fruit,meat`

Using cypress I write the following code

```javascript
cy.request({
            method: 'GET',
            url: '/api/products?category=fruit&category=meat',
            headers: {
                'API-KEY': Cypress.env('API-KEY'),
            },
        }).then(response => {
            console.log(response)
        });
 });
```

and in the this other way too

```javascript
cy.request({
            method: 'GET',
            url: '/api/products?category=fruit&category=meat',
            headers: {
                'API-KEY': Cypress.env('API-KEY'),
            },
           qs: {
                category: ['fruit','meat']
           },
        }).then(response => {
            console.log(response)
        });
 });
```
But the function ends fetching the resource against the API as `/api/products?category[]=fruit&category[]=meat`

PD: I was checking [Axios](https://www.npmjs.com/package/axios#features) library and they have a way to solve this using 

```javascript
  // `paramsSerializer` is an optional function in charge of serializing `params`
  // (e.g. https://www.npmjs.com/package/qs, http://api.jquery.com/jquery.param/)
  paramsSerializer: function (params) {
    return Qs.stringify(params, {arrayFormat: 'repeat'})
  },
```

Is there a way to change this behavior? or any workaround to solve it?

Thanks in advance

**Comment by bahmutov** [comment]
A possible workaround would be to use cy task and make the request from the plug-in file where you can use any way you want to make the request bypassing the cy.request command

**Comment by jonyr** [comment]
@bahmutov Thanks you again. I've found a workaround for this.

I shortened the parameters in the example for practical reasons, but to be hones I need to pass almost 25 query params.

When you told me yesterday to try putting the parameters in the url I just did that with the ones that were arrays, leaving the rest as an object in qs. So it was my mistake. If I pass al the query params in the url the method works as I expected.

Regards,
